.github/actions/spelling/expect.txt

```diff
@@ -11,6 +11,7 @@ adml
 admx
 AFAIK
 aicli
+arning
```

Comment:
This can be avoided by changing:
https://github.com/microsoft/winget-cli/blob/d5a5dce3479943db3bdb4f006d80f21ddc8cf344/.github/actions/spelling/patterns.txt#L80

```diff
- (?:^|[\t ,"'`=(])-[W](?!ait|in|orking) 
+ (?:^|[\t ,"'`=(])-[W](?!ait|arning|in|orking) 
```

Reply:
Thanks for showing me this. Changed to suggested and removed entries from expect.txt